doc/book/src/patterns/id-pointer.md

```diff
@@ -0,0 +1,19 @@
+# ID Pointer
+
+ID Pointer is a technique which allows separation of the main data (an object) and its accessors / capabilities by linking the latter to the original. There's a few different directions in which this pattern can be used:
+
+- issuing transferable capabilities for shared objects (for example, a TransferCap that changes 'owner' field of a shared object)
+- splitting dynamic data and static (eg NFT and its Collection information)
+- avoiding unnecessary type linking (and witness requirement) in generic applications (LP token for a LiquidityPool)
+
+```move
+{{#include ../../examples/sources/patterns/id_pointer.move:4:}}
+```
+
+This pattern is used in these examples:
```

Comment:
```suggestion
The following examples demonstrate using this pattern:
```

Reply:
I'll keep it as is to align with everything else (on every other page we have this wording).